app/Services/ReservationReassignmentService.php

```
@@ -187,6 +187,16 @@ public function reassignOnNewCopy(int $libroId, int $newCopiaId): void
             }
             $reservation = $lockedReservation;
 
+            // Existing same-title duplicates must keep distinct physical copies
+            // even when their windows do not overlap (and even if the setting was
+            // subsequently disabled). Lock sibling commitments before the copy.
+            $committedCopyIds = (new \App\Support\LoanMultiplicityPolicy($this->db))
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '193', column '38'). (undefined)

(MissingImport)